UnityProject/Assets/Scripts/UI/Systems/CameraOverlays/OverlayCrits.cs

```diff
@@ -22,7 +22,7 @@ public class OverlayCrits : MonoBehaviour
 
 		public float Epow = 2;
 
-		private float expectedRadius;
+		public float expectedRadius;
```

Comment:
```suggestion
		public float ExpectedRadius;
```

Reply:
nah, i set this to public while trying to get the flash effect working so i reverted this now.